python/whylogs/api/logger/__init__.py

```diff
@@ -43,6 +43,7 @@ def log(
     obj: Any = None,
     *,
     pandas: Optional[pd.DataFrame] = None,
+    polars: Optional[pl.DataFrame] = None,
```

Comment:
In whylogs 1.x we can't change the name of the `pandas` argument. If we want to support additional flavors of data frames, I'd change the `polars` argument to `dataframe: Optional[Union[pd.DataFrame, pl.DataFrame, ..., DataFrameWrapper]]`.